dev-python/entrypoints/files/entrypoints-0.3-setup.py.patch

```diff
@@ -0,0 +1,20 @@
+setup.py for entrypoints.
```

Comment:
Why is this not being supplied by upstream though?

Reply:
The upstream repository just does not provide a setup.py. Anyway, we could switch over and download the archive from pypi instead of github, as the archive there contains an setup.py generated by flit. Cannot remember if there was anything against this before.

Reply:
Please do it. It means that the automatically produced byproducts over at github are not meant as real release tarballs. Upstreams scripts are obviously adding stuff such as the missing setup.py.